mne/io/egi/egimff.py

```diff
@@ -272,9 +275,12 @@ def _read_locs(filepath, chs, egi_info):
     ch_pos = OrderedDict()
     hsp = list()
     nlr = dict()
+    ch_names = list()
     for sensor in sensors:
         name_element = sensor.getElementsByTagName('name')[0].firstChild
-        name = '' if name_element is None else name_element.data
+        num_element = sensor.getElementsByTagName('number')[0].firstChild
+        name = (channel_naming % int(num_element.data) if name_element is None
+                else name_element.data)
         nr = sensor.getElementsByTagName('number')[0].firstChild.data.encode()
```

Comment:
`nr` and `num_element` are quite similar, this should probably be simplified to
```suggestion
        nr = num_element.data.encode()
```